src/index.html

```diff
@@ -22,6 +22,359 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo of Moyo"
+        />
+      </a>
+      <nav class="navbar">
+        <ul class="navbar__ul">
+          <li class="navbar__li">
```

Comment:
a class should describe what it styles, not repeat tags
```suggestion
        <ul class="navbar__list">
          <li class="navbar__item">
```